presto-native-execution/pom.xml

```diff
@@ -152,4 +152,45 @@
             </plugin>
         </plugins>
     </build>
+
```

Comment:
What are these profiles for ? Why something similar is not needed for the regular Hive testing ?

Reply:
This maven profile separation is needed to allow us configure Glue with AWS credentials. This is very similar how hive tests are already configured. I think it would be better to allow the minimum set of tests use the credentials.